doc/data/messages/d/deprecated-module/pylintrc

```diff
@@ -0,0 +1,2 @@
+[FORMAT]
+deprecated-module = copy
```

Comment:
This need to have a pyversion set or it will start to break when we upgrade the interpreter for the documentation tests. See https://github.com/PyCQA/pylint/blob/main/doc/data/messages/c/continue-in-finally/pylintrc

Reply:
I added a pyversion to the code.